### PR TITLE
Add rules for pushing docs site docker nginx images

### PR DESCRIPTION
**Pull request by lewish**
So we can actually finally serve the new docs site!

### CODE REVIEW

File: docs/BUILD [hunk 1]
```diff
@@ -51,3 +51,23 @@ next_site(
...
+
+load("@io_bazel_rules_docker//container:container.bzl", "container_push")
+
+container_push(
+    name = "push",
+    format = "Docker",
+    image = ":image",
+    registry = "gcr.io",
+    repository = "tada-analytics/static-docs",
+    tag = "latest",
```
Comment (BenBirt) [inline docs/BUILD]: ditto re other PR comment & `latest`

File: scripts/docs/serve [hunk 2]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
```
Comment (BenBirt) [inline scripts/docs/serve]: do we even need this file any longer?

File: docs/BUILD (hunk 1 above)
Comment (lewish) [reply docs/BUILD]: Changed to prod as discussed

File: scripts/docs/serve (hunk 2 above)
Comment (lewish) [reply scripts/docs/serve]: Good point, nah!

**Pull request merged**